src/main/java/org/jenkinsci/plugins/workflow/flow/BlockableResume.java

```diff
@@ -0,0 +1,15 @@
+package org.jenkinsci.plugins.workflow.flow;
+
+/**
+ * Can be added to advertise the ability to mark pipeline components which prevent pipelines from being able to resume
```

Comment:
Unclear from this wording—added to what? What kinds of “components” are being discussed? Is there a specific `add` method you could `link` to in order to help readers understand?